src/app/components/normal-component/chat/chat.component.html

```diff
@@ -1,39 +1,62 @@
-        <ng-template #showChat i18n>Afficher le chat</ng-template> ({{ this.unreadMessages }}<ng-container i18n> nouveau(x) message(s)</ng-container>)
-    </button>
+        <div *ngIf="visible">
+            <button *ngIf="this.showUnreadMessagesButton"
+                id="scrollToBottomIndicator"
+                class="notification button is-info is-light is-fullwidth"
+                (click)="scrollToBottom()"
+                >{{ this.unreadMessagesText }} &darr;</button>
 
-    <form id="chatForm" *ngIf="visible" class="example-form" >
-        <!-- Messages -->
-        <div style="max-height: 90%; overflow: auto">
-            <mat-list *ngIf="chat">
+            <div #chatDiv
+                 id="chatDiv"
+                 class="is-fullwidth block"
+                 style="overflow-y:scroll; height: 5rem;"
+                 (scroll)="updateCurrentScrollPosition()">
                 <p *ngFor="let msg of chat">
                     [{{ msg.postedTime | date:'HH:mm:ss':'+0100' }}] <b>{{ msg.sender }} :</b> {{ msg.content }}
                 </p>
-            </mat-list>
+            </div>
         </div>
 
-        <!-- Text Input -->
-        <mat-form-field class="full-width"
-                        align="start"
-                        *ngIf="userName!=''">
-            <input matInput
-                   #message
-                   maxlength="128"
-                   placeholder="Envoyez un message!"
-                   [(ngModel)]="userMessage"
-                   [ngModelOptions]="{standalone: true}">
-            <mat-icon matSuffix (click)="sendMessage()">send</mat-icon>
-            <mat-hint align="start">
-                <strong i18n>Soyez courtois lol</strong>
-            </mat-hint>
-            <mat-hint align="end">
-                {{message.value.length}} / 128
-            </mat-hint>
-        </mat-form-field>
-    </form>
+        <form id="chatForm" *ngIf="visible"
+              (onSubmit)="sendMessage(message.value)">
+            <div class="field has-addons is-fullwidth">
+                <div class="control is-expanded">
+                    <input class="input"
+                           #message
+                           id="message"
+                           type="text"
+                           placeholder="Entrez votre message ici" i18n-placeholder
+                           maxlength="128"
+                           value=""
+                           [(ngModel)]="userMessage"
+                           [ngModelOptions]="{standalone: true}">
+                    <p class="help">{{ message.value.length }}/128</p>
+                </div>
+                <div class="control">
+                    <button class="button is-primary"
+                            type="submit"
+                            id="send"
+                            (click)="sendMessage(message.value)"
+                            >Send</button>
```

Comment:
close tag at at same x has openning :D ?
Also, are we heretics ? ✝️  :D ?

Reply:
fait pour le close! Pas compris l'autre :D